recipe/meta.yaml

```diff
@@ -12,6 +12,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
+  # skip: True  # [py<36]
+  # noarch: python
```

Comment:
The package will be kept as an architecture specific package for how, however the rest of the code will be kept as a comment so that changes to noarch will be easier in the future.